shared/js/advanced_telemetry_helper.js

```diff
@@ -72,7 +72,7 @@
   ATH.HISTOGRAM_EXP = 0;
   ATH.HISTOGRAM_LINEAR = 1;
   // This is exposed for unit testing purposes
-  ATH.TELEMETRY_ENABLED_KEY = 'debug.performance_data.advanced_telemetry';
+  ATH.TELEMETRY_ENABLED_KEY = 'metrics.selectedMetrics.level';
```

Comment:
Nit: Since the setting is no longer a checkbox, it would be more intuitive for this variable to be a 'level' as opposed to a name representing a bool.